.github/workflows/compose-maven-verify.yaml

```diff
@@ -137,13 +142,16 @@ jobs:
         # yamllint enable rule:line-length
       - name: Build code with Maven
         # yamllint disable rule:line-length
-        uses: Best-Quality-Engineering/ci-maven-action@v1
-        with:
-          phases: ${{ inputs.MVN_PHASES }}
-          pom-file: ${{ inputs.MVN_POM_FILE }}
-          profiles: ${{ inputs.MVN_PROFILES }}
-          maven-opts: ${{ inputs.MVN_OPTS }}
-          settings-file: settings.xml
+        run: |
+          echo "Maven build starting"
+
+          export ${{ inputs.MVN_OPTS }}
+
+          mvn ${{ inputs.MVN_PHASES }} \
+              -e \
+              --global-settings settings.xml \
+              -DaltDeploymentRepository=staging::default::file:"${GITHUB_WORKSPACE}"/m2repo \
+              ${{ inputs.MVN_OPTS }} ${{ inputs.MVN_PARAMS }}
```

Comment:
Related to the above comment, do we need to export the MVN_OPTS to the env and also pass them to the cli command?

Reply:
I will remove the export.. I don't think we need it